design/20221205-memory-management.md

```diff
@@ -310,7 +310,7 @@ Use the new `Secret`s getter in all control loops that need to get any `Secret`:
 
 ### Metrics
 
-The following metrics are based on [a prototype implementation of this design](https://github.com/irbekrm/cert-manager/tree/partial_metadata).
+The following metrics are based on [a prototype implementation of this design](https://github.com/cert-manager/cert-manager/tree/ffe820d310ff2d8bf8efb36ab43b8acd2100be18).
```

Comment:
While the branches exist today, the repo or branches could go away -- it's better to link to commits.